check-dnsseeds.py

```diff
@@ -4,6 +4,22 @@
 Seeds are available from https://github.com/bitcoin/bitcoin/blob/master/src/kernel/chainparams.cpp
 '''
 import subprocess
+from itertools import combinations
+
+NODE_NONE = 0
+NODE_NETWORK = (1 << 0)
+NODE_BLOOM = (1 << 2)
+NODE_WITNESS = (1 << 3)
+NODE_COMPACT_FILTERS = (1 << 6)
+NODE_NETWORK_LIMITED = (1 << 10)
+NODE_P2P_V2 = (1 << 11)
+
+DNS_SEED_PROVIDERS = [
+    "seed.bitcoin.sipa.be.",
```

Comment:
my initial assumption was that we only need to filter for the mainnet  seeds and not the entire list, but i'll make an update to use all the seeds and discard `DNS_SEED_PROVIDERS`

Reply:
It would still be possible to add a flag there to scan or not, i just don't think it needs to be a separate list. With partially overlapping information.

Reply:
> my initial assumption was that we only need to filter for the mainnet  seeds and not the entire list, but i'll make an update to use all the seeds and discard `DNS_SEED_PROVIDERS`